test/io.jl

```diff
@@ -9,7 +9,7 @@
             "fubar" => 50
         )
         roller_prefix = tempname()
-        info("Path to roller_prefix: $roller_prefix")
+        Compat.@info("Path to roller_prefix: $roller_prefix")
```

Comment:
Doesn't have to be `Compat.` anymore since you imported it in test/runtests.jl right?

Reply:
Oh whoops, I missed that one.